src/api/json/catalog.json

```diff
@@ -2803,6 +2803,12 @@
       "fileMatch": ["info.json"],
       "url": "https://raw.githubusercontent.com/Cog-Creators/Red-DiscordBot/V3/develop/schema/red_cog_repo.schema.json"
     },
+    {
+      "name": "Red-DiscordBot Trivia",
+      "description": "Red-DiscordBot Trivia List file",
+      "fileMatch": ["*.yaml", "*.yml"]
```

Comment:
This matches all .yaml and .yml files. It has to be more specific for a fileMatch.

Reply:
Makes sense. It's probably best to remove the fileMatch then - files with this schema can't really be matched based on the file name or location.

Reply:
There's not a specific folder typically used for these files you can use?

Reply:
Custom lists are stored under `cogs/Trivia/*.yaml` in application's data folder though I imagine in most cases one would create/edit those outside that path and later add such custom list through application's interface. As for lists bundled with the application, they're stored under `redbot/cogs/trivia/data/lists/*.yaml` within the application source. I guess we could use both of these to accommodate at least some cases, even if it wouldn't work for all scenarios.